benchbuild/projects/benchbuild/mcrypt.py

```diff
@@ -1,75 +1,84 @@
 from plumbum import local
 
-from benchbuild import project
+import benchbuild as bb
 from benchbuild.settings import CFG
-from benchbuild.utils import compiler, download, path, run, wrapping
+from benchbuild.source import HTTP
+from benchbuild.utils import path
 from benchbuild.utils.cmd import make, tar
 from benchbuild.utils.settings import get_number_of_jobs
 
 
-@download.with_wget({
-    "2.6.8": 'http://sourceforge.net/'
-             'projects/mcrypt/files/MCrypt/2.6.8/mcrypt-2.6.8.tar.gz'
-})
-class MCrypt(project.Project):
+class MCrypt(bb.Project):
     """ MCrypt benchmark """
 
     NAME = 'mcrypt'
     DOMAIN = 'encryption'
     GROUP = 'benchbuild'
-    VERSION = '2.6.8'
-    SRC_FILE = "mcrypt.tar.gz"
+    SOURCE = [
+        HTTP(
+            remote={
+                '2.6.8':
+                    'http://sourceforge.net/projects/mcrypt/files/MCrypt/2.6.8/mcrypt-2.6.8.tar.gz'  # pylint: disable=line-too-long
+            },
+            local='mcrypt.tar.gz'),
+        HTTP(
+            remote={
+                '2.5.8':
+                    'http://sourceforge.net/projects/mcrypt/files/Libmcrypt/2.5.8/libmcrypt-2.5.8.tar.gz'  # pylint: disable=line-too-long
+            },
+            local='libmcrypt.tar.gz'),
+        HTTP(
+            remote={
+                '0.9.9.9':
+                    'http://sourceforge.net/projects/mhash/files/mhash/0.9.9.9/mhash-0.9.9.9.tar.gz'  # pylint: disable=line-too-long
```

Comment:
Codacy found an issue: [Useless suppression of 'line-too-long'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)